mojio_sdk/fuel.py

```diff
@@ -0,0 +1,12 @@
+class Fuel:
+
+    def __init__(self, json_data):
+        self.fuel_level = 0.0
+        self.virtual_fuel_level = 0.0
+        self.fuel_capacity = 0.0
+        if json_data.get('FuelLevel') is not None:
+            self.fuel_level = json_data.get('FuelLevel').get('Value', 0.0)
```

Comment:
I prefer not to call the same getter twice. I'd store the result of `json_data.get('FuelLevel')` in a variable and use it for both statements.